app/main.py

```diff
@@ -1,3 +1,9 @@
+from typing import Counter
+
+
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower:
```

Comment:
The comparison `char.lower() == letter.lower` is incorrect because `letter.lower` is a method reference, not a method call. You need to call the method using parentheses: `letter.lower()`. This will ensure that both `char` and `letter` are compared in lowercase.